test/validators/personal_access_token_validator_test.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class PersonalAccessTokenValidatorTest < ActiveSupport::TestCase
+  setup do
+    @user = build(:user, personal_access_token: '1234')
```

Comment:
For the tests I'm using a `User` model because that was easiest, but if you'd like more separation between `User` and `PersonalAccessTokenValidator` we can create our very own class inside here and use that instead.